packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1207,15 +1207,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     protected async getDataStore(id: string, wait = true): Promise<IFluidRouter> {
-        // Ensure deferred if it doesn't exist which will resolve once the process ID arrives
-        const deferredContext = this.datastoreContexts.ensureDeferred(id);
+        const deferredBind = this.datastoreContexts.prepDeferredBind(id);
```

Comment:
bound vs notBound is only a local concept, since this also works for remote contexts i don't think we should be using the bound terminology here

Reply:
Thanks for clarifying.